container/activities/src/test/java/org/mobicents/slee/runtime/activity/ActivityContextImplTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2011, Red Hat, Inc. and individual contributors
+ * by the @authors tag. See the copyright.txt in the distribution for a
+ * full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.mobicents.slee.runtime.activity;
+
+import javax.slee.resource.ActivityFlags;
+
+import junit.framework.Assert;
+import junit.framework.TestCase;
+
+import org.mobicents.slee.container.activity.ActivityContextCacheDataInterface;
+import org.mobicents.slee.container.activity.ActivityContextHandle;
+import static org.mockito.Mockito.*;
+
+@SuppressWarnings("deprecation")
+public class ActivityContextImplTest extends TestCase {
+
+  public void testGetActivityFlagsNotInitialized() {
+	  ActivityContextHandle ach = mock(ActivityContextHandle.class);
+	  Object obj = null;
+	  ActivityContextCacheDataInterface acCacheHandle = mock(ActivityContextCacheDataInterface.class);
+	  when(acCacheHandle.getObject(obj)).thenReturn(null);

```

Comment:
this is the good thing about mock libs, you specify the expected behavior for collaborators along the same lines of the test,instead of creating fake stub classes, that dont relate to the test.. This is proper BDD (https://martinfowler.com/bliki/GivenWhenThen.html)